.github/workflows/compatibility.yml

```diff
@@ -12,13 +12,13 @@ jobs:
       max-parallel: 10
       matrix:
         # Minimum support version, ..., Latest version, `beta`.
-        woocommerce: [ '6.0.1', '6.1.2', '6.2.2', '6.3.1', '6.4.1', '6.5.1', '6.6.0', 'latest', 'beta' ]
+        woocommerce: [ '6.2.2', '6.3.1', '6.4.1', '6.5.1', '6.6.0', '6.7.0', 'latest', 'beta' ]
```

Comment:
Should `WC 6.6.0` be [`WC 6.6.1`](https://github.com/woocommerce/woocommerce/releases/tag/6.6.1) here?

Reply:
Good catch! Fixed in [094e3f3](https://github.com/Automattic/woocommerce-payments/pull/4524/commits/094e3f3b9008161e6342fc1153ad975e9fee2125)